amd_openvx_extensions/CMakeLists.txt

```diff
@@ -63,7 +79,22 @@ if(GPU_SUPPORT AND OpenCL_FOUND AND BUILD_OPENCL)
         message("-- ${Red}WARNING:FFMPEG Not Found -- amd_media module excluded${ColourReset}")
     endif(FFMPEG_FOUND)
 else()
-    message("-- ${Red}WARNING:OpenCL Not Found -- amd_rpp, amd_loomsl, & amd_media modules excluded${ColourReset}")
+    message("-- ${Red}WARNING:OpenCL/FFMPEGNot Found -- amd_loomsl, & amd_media modules excluded${ColourReset}")
+endif()
+
+if(GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP")
+        if(HIP_FOUND)
+            if(AMDRPP_FOUND)
+                add_subdirectory(amd_rpp)
+            else()
+                message("-- ${Red}WARNING:AMDRPP Not Found -- amd_rpp module excluded${ColourReset}")
+            endif()
+            if(HIP_COMPILER STREQUAL clang)
+                set(hip_library_name amdhip64)
+            endif()
```

Comment:
add the below code in the case clang is not available
else()
message(FATAL_ERROR "unsupported hip compiler")

Reply:
Done.